aip-console-tools-cli/src/test/java/com/castsoftware/aip/console/tools/AddVersionCommandIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package com.castsoftware.aip.console.tools;
+
+import org.assertj.core.util.Arrays;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@RunWith(MockitoJUnitRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("INTEGRATION_PROFILE_TEST")
+public class AddVersionCommandIntegrationTest {
+
+    @Autowired
+    private AipConsoleToolsCommandTest aipConsoleToolsCommandTest;
+
+    @Test
+    public void testAddVersionCommand() {
```

Comment:
this test is to test what?

Reply:
Preparing for the addversion integration test.